src/app/resources/iis_scripts/admin_api_util.ps1

```diff
@@ -0,0 +1,120 @@
+param(
+    [Parameter(Mandatory=$true)]
+    [ValidateSet('install', 'permission-check')]
+    [string]
+    $command,
+
+    [string]
+    $downloadFrom,
+
+    ## Not doing anything with this parameter yet
+    [string]
+    $sessionId,
+
+    [string]
+    $serviceName = "Microsoft IIS Administration",
+
+    [string]
+    $iisAdminGroup = "WAC-IISAdmin"

```

Comment:
The name doesn't necessarily need WAC in it. Also, I would stay away from abbreviations in group names. The members of this group are being added to the "Owners" role within the IIS Administration API, so they will have the highest rights with the API. I would suggest naming it _IIS Administration Owners_.